### PR TITLE
Update milking_update docstring

### LINKED ISSUE #971: Cite `milking_update` equations in docstring
In `cow.py`, the `milking_update` method has a series of equations that are not cited. Docstrings could be updated to include citations from pseudocode.

### PR DESCRIPTION
**Pull request by ktthr**
Issue(s) closed by this pull request: #971 

Other than addressing #971 (adding citations for pseudocode equations to the milking_update docstring), this pull request also generally updates the docstring to adhere to guidelines (reformatting, adding sections, etc.)

### CODE REVIEW

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 1]
```diff
@@ -343,20 +343,42 @@ def determine_param_value(mean, std):
...
-            fat_percent: calculated with days in milk, for temporary use
-            daily_fat_correct_milk_production: calculated form estimated
-                milk production and fat percent, for temporary use
+        Updates milking status for lactating cows, using breed and parity
+        to estimate daily milk production from specific lactation curves
+        (following Wood's curve model.)
+
+        Parameters
+        -------
+        sim_day : Any
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/life_cycle/cow.py]: sim_day should be int, and have the description of something like "The current simulation day."

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 2]
```diff
@@ -343,20 +343,42 @@ def determine_param_value(mean, std):
...
+        calving_interval : Any
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/life_cycle/cow.py]: Here calving interval is either float or int: there's logic in the `_set_avg_CI` method that defines whether it's the int input or the float average. You could add something like that to the description of the `calving_interval` attribute here.

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 3]
```diff
@@ -343,20 +343,42 @@ def determine_param_value(mean, std):
...
+        fat_percent : float
+            Calculated with days in milk, for temporary use
+        daily_fat_correct_milk_production : Any | float
+            Calculated from estimated milk production and fat percent, for temporary use
+
+        Notes
+        ------
+        - The targeted lactation curve is set at the beginning of each lactation,
+        and the variance can be added after adjusted by the nutrition submodel.
+        - TEMP: Currently, the variables for fat percent, FCM,
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/life_cycle/cow.py]: I'd remove the `TEMP:` part of the second Note here, just to tidy up the old text a little more.

**Review by JoeWaddell — COMMENTED** [review]
Nice work! A few small changes/updates and it'll be good to go.

**Comment by ktthr** [comment]
> Nice work! A few small changes/updates and it'll be good to go.

@JoeWaddell Hi Joe, I've just added these changes into the branch.

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 4]
```diff
@@ -343,20 +343,47 @@ def determine_param_value(mean, std):
...
+            and is True; otherwise, it is a float (average herd calving
+            interval) (logic in _set_avg_CI in of lifecycle.py)
+
+        Returns
+        -------
+        estimated_daily_milk_produced : float
+            Estimated daily milk production from the lactation curve
+        fat_percent : float
+            Calculated with days in milk, for temporary use
+        daily_fat_correct_milk_production : Any | float
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
        daily_fat_correct_milk_production : float
```

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 5]
```diff
@@ -343,20 +343,47 @@ def determine_param_value(mean, std):
...
-                milk production and fat percent, for temporary use
+        Updates milking status for lactating cows, using breed and parity
+        to estimate daily milk production from specific lactation curves
+        (following Wood's curve model.)
+
+        Parameters
+        -------
+        sim_day : int
+            The current simulation day
+        calving_interval : int or float
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
        calving_interval : int | float
```

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 6]
```diff
@@ -343,20 +343,47 @@ def determine_param_value(mean, std):
...
-            daily_fat_correct_milk_production: calculated form estimated
-                milk production and fat percent, for temporary use
+        Updates milking status for lactating cows, using breed and parity
+        to estimate daily milk production from specific lactation curves
+        (following Wood's curve model.)
+
+        Parameters
+        -------
+        sim_day : int
+            The current simulation day
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
            The current simulation day.
```

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 7]
```diff
@@ -343,20 +343,47 @@ def determine_param_value(mean, std):
...
+        calving_interval : int or float
+            calving_interval is an int (user input calving interval) if the
+            value of animal_config["use_input_calving_interval"] exists
+            and is True; otherwise, it is a float (average herd calving
+            interval) (logic in _set_avg_CI in of lifecycle.py)
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
            interval), see logic in _set_avg_CI in lifecycle.py.
```

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 8]
```diff
@@ -343,20 +343,47 @@ def determine_param_value(mean, std):
...
+
+        Returns
+        -------
+        estimated_daily_milk_produced : float
+            Estimated daily milk production from the lactation curve
+        fat_percent : float
+            Calculated with days in milk, for temporary use
+        daily_fat_correct_milk_production : Any | float
+            Calculated from estimated milk production and fat percent, for temporary use
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
            Calculated from estimated milk production and fat percent, for temporary use.
```

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 9]
```diff
@@ -343,20 +343,47 @@ def determine_param_value(mean, std):
 
     def milking_update(self, sim_day, calving_interval):
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
    def milking_update(self, sim_day: int, calving_interval: int | float):
```

**Review by JoeWaddell — APPROVED** [review]
Just a few small suggestions, nice work!

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 10]
```diff
@@ -341,22 +341,49 @@ def determine_param_value(mean, std):
...
-            estimated_daily_milk_produced: estimated daily milk production
-                from the lactation curve
-            fat_percent: calculated with days in milk, for temporary use
-            daily_fat_correct_milk_production: calculated form estimated
-                milk production and fat percent, for temporary use
+    def milking_update(self, sim_day: int, calving_interval: int | float):
+        """
+        Updates milking status for lactating cows, using breed and parity
+        to estimate daily milk production from specific lactation curves
+        (following Wood's curve model.)
```
Comment (ew3361zh) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
        (following Wood's curve model).
```

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 11]
```diff
@@ -341,22 +341,49 @@ def determine_param_value(mean, std):
...
+        calving_interval : int | float
+            calving_interval is an int (user input calving interval) if the
+            value of animal_config["use_input_calving_interval"] exists
+            and is True; otherwise, it is a float (average herd calving
+            interval), see logic in _set_avg_CI in lifecycle.py.
+
+        Returns
+        -------
+        estimated_daily_milk_produced : float
+            Estimated daily milk production from the lactation curve
```
Comment (ew3361zh) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
            Estimated daily milk production from the lactation curve.
```

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 12]
```diff
@@ -341,22 +341,49 @@ def determine_param_value(mean, std):
...
+
+        Parameters
+        -------
+        sim_day : int
+            The current simulation day.
+        calving_interval : int | float
+            calving_interval is an int (user input calving interval) if the
+            value of animal_config["use_input_calving_interval"] exists
+            and is True; otherwise, it is a float (average herd calving
+            interval), see logic in _set_avg_CI in lifecycle.py.
```
Comment (ew3361zh) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
            interval). See logic in _set_avg_CI in lifecycle.py.
```

File: RUFAS/routines/animal/life_cycle/cow.py [hunk 13]
```diff
@@ -341,22 +341,49 @@ def determine_param_value(mean, std):
...
+
+        Returns
+        -------
+        estimated_daily_milk_produced : float
+            Estimated daily milk production from the lactation curve
+        fat_percent : float
+            Calculated with days in milk, for temporary use
```
Comment (ew3361zh) [inline RUFAS/routines/animal/life_cycle/cow.py]:
```suggestion
            Calculated with days in milk, for temporary use.
```

**Review by ew3361zh — APPROVED** [review]
LGTM! Just a few small grammatical suggestions to review before merging. Nice work!

**Pull request merged**